packages/@aws-cdk/custom-resources/README.md

```diff
@@ -148,28 +150,25 @@ The return value must be a JSON object with the following fields:
 Every resource in CloudFormation has a physical resource ID. When a resource is
 created, the `PhysicalResourceId` returned from the `Create` operation is stored
 by AWS CloudFormation and assigned to the logical ID defined for this resource
-in the template.
+in the template. If a `Create` operation returns without a `PhysicalResourceId`,
+the framework will use `RequestId` as the default. This may be sufficient for
```

Comment:
"may be" == weasel words. Either it is or it isn't. In this case I would agree that it is.

(Probably :stuck_out_tongue_closed_eyes: )

Reply:
updated.